.github/workflows/pre-commit-hooks.yml

```diff
@@ -12,6 +12,6 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v4
         with:
-          python-version: '3.x'
+          python-version: '3.8'
```

Comment:
Please undo this. I want to be alerted of any problems with the latest version of python, even if that is not what I use locally,

If this has something to do with the specific version required by clang tools, then it should be addressed in a CI that actually uses the clang tools.

Reply:
Well, python v3.11 was released today. I don't see anything in the [3.11 changelog](https://docs.python.org/3/whatsnew/3.11.html) related to the errors the failed CI logs. I also checked with the [pylint changelog](https://github.com/PyCQA/pylint/releases) and I don't see anything related to removing the `missing-timeout` check.

I suppose we can avoid testing with python v3.11 or newer while they work out the "day 1" bugs.